Riot/ViewController/ReadReceiptsViewController.m

```diff
@@ -0,0 +1,161 @@
+    
+    [self configureViews];
+    [self configureReceiptsTableView];
+    [self addOverlayViewGesture];
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Views
+
+- (void)configureViews
+{
+    self.containerView.layer.cornerRadius = 20;
+    self.titleLabel.text = @"Read Receipts List";
+}
+
+- (void)configureReceiptsTableView
+{
+    self.receiptsTableView.dataSource = self;
+    self.receiptsTableView.delegate = self;
+    self.receiptsTableView.showsVerticalScrollIndicator = NO;
+    self.receiptsTableView.separatorStyle = UITableViewCellSeparatorStyleNone;
+    
+    [self.receiptsTableView registerNib:[MXKReadReceiptTableViewCell nib] forCellReuseIdentifier:[MXKReadReceiptTableViewCell defaultReuseIdentifier]];
+}
+
+- (void)addOverlayViewGesture
+{
+    UITapGestureRecognizer *tapRecognizer = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(overlayTap)];
+    [tapRecognizer setNumberOfTapsRequired:1];
+    [tapRecognizer setNumberOfTouchesRequired:1];
+    [self.overlayView addGestureRecognizer:tapRecognizer];
+}
+
+#pragma mark - Actions
+
+- (void)overlayTap
+{
+    [self dismissViewControllerAnimated:YES completion:nil];
+}
+
+#pragma mark - UITableViewDataSource
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
+{
+    return self.roomMembers.count;
+}
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
+{
+    return 1;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    MXKReadReceiptTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:[MXKReadReceiptTableViewCell defaultReuseIdentifier] forIndexPath:indexPath];
+    
+    if (indexPath.row < self.roomMembers.count)
+    {
+        NSString *name = self.roomMembers[indexPath.row].displayname;
+        if (name.length == 0) {
+            name = self.roomMembers[indexPath.row].userId;
+        }
+        cell.displayNameLabel.text = name;
+    }
+    if (indexPath.row < self.placeholders.count)
+    {
+        NSString *avatarUrl = self.roomMembers[indexPath.row].avatarUrl;
+        if (self.restClient && avatarUrl)
+        {
+            CGFloat side = CGRectGetWidth(cell.avatarImageView.frame);
+            avatarUrl = [self.restClient urlOfContentThumbnail:avatarUrl toFitViewSize:CGSizeMake(side, side) withMethod:MXThumbnailingMethodCrop];
+        }
+        [cell.avatarImageView setImageURL:avatarUrl withType:nil andImageOrientation:UIImageOrientationUp previewImage:self.placeholders[indexPath.row]];
+    }
+    if (indexPath.row < self.receipts.count)
+    {
+        NSString *receiptDescription = [(MXKEventFormatter*)self.session.roomSummaryUpdateDelegate dateStringFromTimestamp:self.receipts[indexPath.row].ts withTime:YES];
+        cell.receiptDescriptionLabel.text = receiptDescription;
```

Comment:
You may use attributed string to include the prefix "Read:" in this description.
(see https://github.com/matrix-org/matrix-ios-kit/blob/master/MatrixKit/Views/DeviceView/MXKDeviceView.m#L129 for an example of NSMutableAttributedString use)